13.go

```diff
@@ -32,14 +33,22 @@ const numSessionTickets = 2
 type secretLabel int
 
 const (
-	x25519SharedSecretSz = 32
-
-	P503PubKeySz                  = 378
-	P503PrvKeySz                  = 32
-	P503SharedSecretSz            = 126
-	SIDHp503Curve25519PubKeySz    = x25519SharedSecretSz + P503PubKeySz
-	SIDHp503Curve25519PrvKeySz    = x25519SharedSecretSz + P503PrvKeySz
-	SIDHp503Curve25519SharedKeySz = x25519SharedSecretSz + P503SharedSecretSz
+	// Both public key and shared secret size
+	x25519Sz = 32
+
+	SIDHp503PubKeySz              = 378
+	SIDHp503PrvKeySz              = 32
```

Comment:
SIKE and SIDH private key size is different. Not sure where in documentation it is but
```
     378 : Public Key
   +  32 : "real" random private key - as defied for SIDH (252 bits for Allice and 250 for Bob in case of P503)
   +  24 : Message (see comment below)
---------
      434
```